posthog/hogql/database/schema/persons_pdi.py

```diff
@@ -85,11 +91,7 @@ def persons_pdi_join(
 # :NOTE: We already have person_distinct_ids.py, which most tables link to. This persons_pdi.py is a hack to
 # make "select persons.pdi.distinct_id from persons" work while avoiding circular imports. Don't use directly.
 class PersonsPDITable(LazyTable):
-    fields: dict[str, FieldOrTable] = {
-        "team_id": IntegerDatabaseField(name="team_id", nullable=False),
-        "distinct_id": StringDatabaseField(name="distinct_id", nullable=False),
-        "person_id": StringDatabaseField(name="person_id", nullable=False),
-    }
+    fields: dict[str, FieldOrTable] = FIELDS
```

Comment:
Hmm, I'm not sure about this

We mutate several tables' fields during `create_hogql_database`

If every HogQL DB uses a shared field object, it will pick up changes from another `create_hogql_database` call with different modifiers. This would be a nightmare to debug, and more likely to show up in prod than in testing.